scripts/install.sh

```diff
@@ -9,6 +9,9 @@ DONE_END='\n\n\e[0m'    # new line + reset
 KEEP_CORE_PATH=$PWD
 
 KEEP_BEACON_SOL_PATH="$KEEP_CORE_PATH/solidity/random-beacon"
+KEEP_ECDSA_SOL_PATH="$KEEP_CORE_PATH/solidity/ecdsa"
+TBTC_PATH="$KEEP_CORE_PATH/tbtc-v2"
```

Comment:
I am confused, why do we set `TBTC_PATH` to this value?

Reply:
Yea, this guy can be removed, probably a leftover from the first attempts of dealing with `tbtc-v2` [18da419](https://github.com/keep-network/keep-core/pull/3096/commits/18da419888a0cf2f9697d9599120de41a6056205)